x/gov/client/cli/query.go

```diff
@@ -226,21 +235,30 @@ $ gaiacli query gov votes 1
 				return fmt.Errorf("proposal-id %s not a valid int, please input a valid proposal-id", args[0])
 			}
 
+			params := gov.NewQueryProposalParams(proposalID)
+			bz, err := cdc.MarshalJSON(params)
+			if err != nil {
+				return err
+			}
+
 			// check to see if the proposal is in the store
-			_, err = queryProposal(proposalID, cliCtx, cdc, queryRoute)
+			res, err := queryProposal(proposalID, cliCtx, cdc, queryRoute)
 			if err != nil {
 				return fmt.Errorf("Failed to fetch proposal-id %d: %s", proposalID, err)
 			}
 
-			// Construct query
-			params := gov.NewQueryProposalParams(proposalID)
-			bz, err := cdc.MarshalJSON(params)
-			if err != nil {
+			var proposal gov.Proposal
+			if err := cdc.UnmarshalJSON(res, &proposal); err != nil {
 				return err
 			}
 
-			// Query store
-			res, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/votes", queryRoute), bz)
+			propStatus := proposal.GetStatus()
+			if !(propStatus == gov.StatusVotingPeriod || propStatus == gov.StatusDepositPeriod) {
```

Comment:
this will always return an empty tally on the deposit period. Better just check if it’s not voting period

Reply:
I think we need both checks. I had voting period only at first and the LCD test was failing because it was returning not votes (same with deposits).

Reply:
why was it querying the deposits ? as long as it's in voting period there should be at least one deposit